dali/operators/image/resize/resize_attr.cc

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/resize/resize_attr.h"
+#include <limits>
+#include <string>
+#include <vector>
+#include "dali/pipeline/operator/common.h"
+#include "dali/core/math_util.h"
+
+namespace dali {
+
+DALI_SCHEMA(ResizeAttr)
+  .AddOptionalArg("resize_x", R"(The length of the X dimension of the resized image.
+This option is mutually exclusive with `resize_shorter`, `resize_longer` and `size`.
+If the `resize_y` is left unspecified or 0, then the op will keep
+the aspect ratio of the original image. Negative value flips the image.)", 0.f, true)
+  .AddOptionalArg("resize_y", R"(The length of the Y dimension of the resized image.
+This option is mutually exclusive with `resize_shorter`, `resize_longer` and `size`.
+If the `resize_x` is left unspecified or 0, then the op will keep
```

Comment:
Aren't we suppose to use "``", not "`"`?

Reply:
Will fix.